Dockerfile

```diff
@@ -19,9 +19,11 @@ COPY config/ config/
 COPY hack/ hack/
 
 ARG VERSION
+ARG TARGETOS
+ARG TARGETARCH
```

Comment:
Shouldn't the Makefile be modified as well?

Reply:
@igor-karpukhin If you see the ENVs below, it gets the value of these arguments and the envs are used in the Makefile to build for proper OS/ARCH. With that, no changes are needed in it.